src/connectors/status/status-db.connector.test.js

```diff
@@ -54,6 +55,62 @@ describe("Function: getStoredStatus", () => {
     });
   });
 
+  describe("When: topology is invalid", () => {
```

Comment:
given that I didn't update the tests in front-end, should we not have had coverage <100%? I guess it must cover but just not test.